src/renderer/components/Delegation/ValidatorRow.js

```diff
@@ -0,0 +1,246 @@
+  position: relative;
+  overflow: hidden;
+  border-color: ${p =>
+    p.active ? p.theme.colors.palette.primary.main : p.theme.colors.palette.divider};
+  ${p =>
+    p.active
+      ? `&:before {
+        content: "";
+        width: 4px;
+        height: 100%;
+        top: 0;
+        left: 0;
+        position: absolute;
+        background-color: ${p.theme.colors.palette.primary.main};
+      }`
+      : ""}
+  ${p =>
+    p.disabled
+      ? css`
+          ${InputBox} {
+            pointer-events: none;
+          }
+        `
+      : ""}
+`;
+
+export const IconContainer: ThemedComponent<*> = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 24px;
+  height: 24px;
+  border-radius: 4px;
+  background-color: ${p =>
+    p.isSR ? p.theme.colors.palette.action.hover : p.theme.colors.palette.divider};
+  color: ${p =>
+    p.isSR ? p.theme.colors.palette.primary.main : p.theme.colors.palette.text.shade60};
+`;
+
+const InfoContainer = styled(Box).attrs(() => ({
+  vertical: true,
+  ml: 2,
+  flex: 1,
+}))``;
+
+const Title = styled(Box).attrs(() => ({
+  horizontal: true,
+  alignItems: "center",
+}))`
+  width: min-content;
+  max-width: 100%;
+  font-size: 13px;
+  font-weight: 600;
+  cursor: pointer;
+  color: ${p => p.theme.colors.palette.text.shade100};
+  ${IconContainer} {
+    background-color: rgba(0, 0, 0, 0);
+    color: ${p => p.theme.colors.palette.primary.main};
+    opacity: 0;
+  }
+  &:hover {
+    color: ${p => p.theme.colors.palette.primary.main};
+  }
+  &:hover > ${IconContainer} {
+    opacity: 1;
+  }
+  ${Text} {
+    flex: 0 1 auto;
+    display: block;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+`;
+
+const SubTitle = styled(Box).attrs(() => ({
+  horizontal: true,
+}))`
+  font-size: 11px;
+  font-weight: 500;
+  color: ${p => p.theme.colors.palette.text.shade60};
+`;
+
+const RightFloating = styled.div`
+  position: absolute;
+  right: 0;
+  padding: 8px;
+  pointer-events none;
+  opacity: 0;
+`;
+
+const InputBox = styled(Box).attrs(() => ({
+  horizontal: true,
+  alignItems: "center",
+}))`
+  position: relative;
+  flex-basis: 150px;
+  height: 32px;
+  &:focus > ${RightFloating}, &:focus-within > ${RightFloating} {
+    opacity: 1;
+    pointer-events: auto;
+  }
+`;
+
+const VoteInput = styled.input.attrs(() => ({
+  type: "text",
+  step: 1,
+  min: 0,
+  pattern: "[0-9]",
+  placeholder: 0,
+}))`
+  cursor: pointer;
+  flex: 1;
+  text-align: left;
+  font-size: 13px;
+  font-weight: 600;
+  height: 100%;
+  padding: 0 8px;
+  background-color: rgba(0, 0, 0, 0);
+  border: 1px solid transparent;
+  border-radius: 4px;
+  border-color: ${p => (p.notEnoughVotes ? p.theme.colors.pearl : p.theme.colors.palette.divider)};
+  &:disabled {
+    cursor: disabled;
+    color: ${p => p.theme.colors.palette.text.shade40};
+    background-color: ${p => p.theme.colors.palette.background.default};
+  }
+`;
```

Comment:
I don't 100% understand why we always need to redefine input style again and again. Don't we have an Input component?